tests/steamship_tests/utils/deployables.py

```diff
@@ -60,7 +60,7 @@ def zip_deployable(file_path: Path) -> bytes:
                 "idna",
                 "pydantic==1.9.0",
                 "typing_extensions",
-                "inflection==0.5.1",
+                "inflection",
```

Comment:
Not high priority --- I wonder if we could eventually replace this with reading in `requirements.txt` & the `.dev.txt` versions so that we guarantee alignment